src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -648,6 +719,12 @@ ucs_status_t uct_dc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
         goto out;
     }
 
+    if ((flags & UCT_FLUSH_FLAG_REMOTE) &&
+        (ep->flags & UCT_DC_MLX5_EP_FLAG_FLUSH_REMOTE_ENABLED) &&
+        (ep->flags & UCT_DC_MLX5_EP_FLAG_FLUSH_REMOTE)) {
```

Comment:
maybe it's possible to unite these 2 flags somehow?

Reply:
> maybe it's possible to unite these 2 flags somehow?

no: we have to store 2 states, no ideas how to store it into single flag

Reply:
use test_all_flags?